drivers/media/i2c/imx258.c

```diff
@@ -28,7 +28,7 @@
 #define IMX258_VTS_30FPS		0x0c50
 #define IMX258_VTS_30FPS_2K		0x0638
 #define IMX258_VTS_30FPS_VGA		0x034c
-#define IMX258_VTS_MAX			0xffff
+#define IMX258_VTS_MAX			65525
```

Comment:
In an earlier commit you have:
```diff
+#define IMX258_EXPOSURE_OFFSET		10
...
-#define IMX258_EXPOSURE_MAX		65535
+#define IMX258_EXPOSURE_MAX		(IMX258_VTS_MAX - IMX258_EXPOSURE_OFFSET)
```
giving an IMX258_EXPOSURE_MAX of 65525. This commit changes it to 65515 - is this correct?

Reply:
Yes. The original driver was wrong twice over.

The datasheet lists the range of 0x340/1 FRM_LENGTH_LINES to be <=65525. (FWIW table 4-4)

Section 9.3 then states
> The setting range of exposure time during long exposure mode is as follows:
> ```1 <= COARSE_INTEG_TIME <= FRM_LENGTH_LINES - 10```

Also documented under register 0x0350 FRM_LENGTH_CTL as the alpha parameter in a calculation, "type specific adjustment parameter, and is 10(d) for this type"